.github/workflows/e2e-tests.yml

```diff
@@ -94,6 +94,21 @@ jobs:
           sed -i -e 's/\"istanbul\",//g' .babelrc
           sed -i -e 's/\"lodash\",/\"lodash\", \"istanbul\",/g' .babelrc
 
+      - name: Configure AWS Credentials
+        if: ${{ env.BUILDTYPE == 'vagovprod' }}
```

Comment:
You don't need this `if`clause here since this workflow only builds prod

Reply:
Makes sense. Thanks!